pkg/features/kube_features.go

```diff
@@ -638,6 +638,12 @@ const (
 	//
 	// Disables Accelerator Metrics Collected by Kubelet
 	DisableAcceleratorUsageMetrics featuregate.Feature = "DisableAcceleratorUsageMetrics"
+
+	// owner: @roycaihw
+	// alpha: v1.20
+	//
+	// Assigns each kube-apiserver an ID in a cluster.
```

Comment:
nit: in a HA cluster?

Reply:
Although it's mostly a feature for HA clusters, I omitted HA because the ID also gets created in non-HA clusters as well. WDYT?